ltr/concourse/deploy.py

```diff
@@ -40,13 +46,16 @@
     print("could not find current endpoint, will create a new one", file=sys.stderr)
 
 # find the model
+model_location = model_tag if model_tag != "" else model_name
 model_keys = session.boto_session.client("s3").list_objects_v2(
-    Bucket=s3_bucket, Prefix=f"model/{model_name}"
+    Bucket=s3_bucket, Prefix=f"model/{model_location}"
```

Comment:
You could even put the `model_location` definition in here if you wanted, making this expression:

```
model_keys = session.boto_session.client("s3").list_objects_v2(
    Bucket=s3_bucket, Prefix=f"model/{model_tag if model_tag else model_name}"
)["Contents"]
```

Reply:
I want to print the location later so will keep it separate for now.